### DISCUSSION TITLE
How do you maintain RLS in a ci/cd fashion?

### DISCUSSION THREAD

**Discussion by stunaz** [supabase/supabase#13652] [Questions] [answered]
Hello, might be a noob question. But how do you maintain your RLS policies ?
I was thinking about versioning them in my source code, then somehow executing in my ci/cd workflow (GitHub-actions).

Perhaps, you can give an idea on how you mange them.

please dont say terraform :-)

**Comment by sweatybridge** [comment, marked as answer]
Committing RLS policies in a `supabase/migrations` file would be the way to go. The migrations can be pushed to your project using [setup-cli](https://github.com/supabase/setup-cli?tab=readme-ov-file#advanced-usage) GHA.
